o11y/metrics/datadog/datadog.go

```diff
@@ -184,9 +184,9 @@ func (d Sink) MetricOrphanFound(tags []string) error {
 	return d.metricWithStatus(metricPrefixController+"orphan.found", tags)
 }
 
-// MetricCacheTriggered signals a selector cache trigger
-func (d Sink) MetricSelectorCacheTriggered(tags []string) error {
-	return d.metricWithStatus(metricPrefixController+"selector.cache.triggered", tags)
+// MetricWatcherCalls is a counter of watcher calls.
+func (d Sink) MetricWatcherCalls(tags []string) error {
+	return d.metricWithStatus(metricPrefixController+"watcher.calls", tags)
```

Comment:
IMO this metric is emitted as a singular event eg. `MetricWatcherCall`, `"watcher.call"`, or `called` etc.